packages/wallet/src/remote-signers/callback-remote-signer.ts

```diff
@@ -0,0 +1,19 @@
+import { BytesLike } from 'ethers'
+import { ChainIdLike } from '@0xsequence/network'
+import { RemoteSigner } from './remote-signer'
+
+export type SignCallback = (message: BytesLike, data?: BytesLike, chainId?: ChainIdLike) => Promise<string>
+
+export class CallbackRemoteSigner extends RemoteSigner {
```

Comment:
similar to above comment, we should think about the right name for this.. and make sure it `implements Signing` or whatever we name the interface above